jest.config.js

```diff
@@ -10,9 +10,10 @@ const config = {
     }
   },
   testEnvironment: 'node',
-  testMatch: ['**/test/sitesupport/**/*.test.js'],
-  testPathIgnorePatterns: ['/node_modules/'],
+  testMatch: [ '**/test/sitesupport/**/*.test.js' ],
+  testPathIgnorePatterns: [ '/node_modules/' ],
   verbose: true
-}
+};
 
-module.exports = config;
\ No newline at end of file
+// eslint-disable-next-line no-undef
+module.exports = config;
```

Comment:
Some things added in **Add jest** that would be ideally added correctly there instead of fixed here.